docs/LAUNCHING.md

```diff
@@ -0,0 +1,7 @@
+# Launching your extension to production
+
+**Note:** Currently, you need to publish a version of your extension before running it on a Shopify store.
+
+Learn how to [test your extension on a Shopify store](./TESTING.md#testing-on-shop).
+
+To test changes without changing the live version of your extension, add it to a test version of your app and publish there. Once your changes are ready, copy the Argo code to the extension on your live app.
```

Comment:
With this new flow this isn't really true, right? They can perform tests on their app extensions without affecting live. Should we not be specific about that? 

E.g. `Some app extensions, such as those that support Argo, now offer the ability to test changes in development stores without making changes to production or creating a copy of your app. Learn more about how to test your extensions.`